src/bindings/python/fluxacct/accounting/project_subcommands.py

```diff
@@ -12,6 +12,34 @@
 import sqlite3
 
 
+###############################################################
+#                                                             #
+#                      Helper Functions                       #
+#                                                             #
+###############################################################
+
+
+# check if bank already exists and is active in bank_table;
```

Comment:
From the department of trivial things, this comment should probably refer to 'project' and 'project_table' rather than 'bank' and 'bank_table'.

Reply:
Whoops, you're totally right. 🤦 Thanks for catching this. Just force-pushed a fix to this comment.